client/components/Weather.jsx

```diff
@@ -1,8 +1,24 @@
+import ENV from '../../map.env';
 import React from 'react';
 import axios from 'axios';
 import { useEffect, useState } from "react";
 
+// const sdk = require('api')('@climacell-docs/v4#bzqumkyn13ppf');
+
+
+
 const Weather = () => {
+  const [temp, setTemp] = useState(0);
+  let lat = 29.9430
+  let lon = -90.3517
+  const getWeather = () => {
+    axios.get(`https://api.openweathermap.org/data/3.0/onecall?lat=${lat}&lon=${lon}&units=imperial&lang=en&exclude=minutely,hourly,alerts&appid=${ENV.WEATHER}`)

```

Comment:
instead of making the call from the frontend to the API, you should make a call to your server, which then makes a call to the API to hide your sensitive env data

Reply:
ah that makes sense. thank you!